sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/logging/HttpRequestLoggingInterceptor.java

```diff
@@ -40,9 +34,11 @@ public void afterCompletion(
         var remoteAddressIgnored = MDC.putCloseable("remoteAddress", request.getRemoteAddr());
         var statusIgnored = MDC.putCloseable("status", String.valueOf(response.getStatus()));
         var requestTraceTokenIgnored =
-            MDC.putCloseable("requestHeaderTraceToken", request.getHeader("trace-token"));
+            MDC.putCloseable(
```

Comment:
don't we want to use the same MDC value [like in sda-dw-commons](https://github.com/SDA-SE/sda-dropwizard-commons/blob/3980132675cbe518b58ff48ca8daaff20f5ac608/sda-commons-shared-tracing/src/main/java/org/sdase/commons/shared/tracing/RequestTracing.java#L19), i.e. `Trace-Token`?

Reply:
`MDC` only supports String values. In Dropwizard, the request logging is not restricted by SLF4J and therefore able to build a completely custom Json. There we have a Map for request headers and a map for response headers. but that is not possible here.